Apps/Playground/iOS/LibNativeBridge.mm

```diff
@@ -56,7 +59,10 @@ - (void)init:(void*)CALayerPtr width:(int)inWidth height:(int)inHeight
     Babylon::InitializeNativeEngine(*runtime, windowPtr, width, height);
     
     // Initialize XMLHttpRequest plugin.
-    Babylon::InitializeXMLHttpRequest(*runtime, runtime->RootUrl());
+    runtime->Dispatch([](Napi::Env env)
+    {
+        InitializeXMLHttpRequest(env);
+    });
```

Comment:
I think we want to move away from this pattern and more toward just passing the runtime itself to the plugin initializers, which can then do their own dispatching. Creating a new pattern for this is really a part of the post-plugin refactor cleanup work, which hasn't been done yet, but that's the direction I think it'll go.